core/sql/regress/charsets/EXPECTED012

```diff
@@ -120,11 +121,11 @@ tomorrow_suffix
 
```

Comment:
I take it the character set synthesis rules changed somewhere? The new results seem to be twice as long as the old.